docs/onboarding/custom-framework-overview.md

```diff
@@ -0,0 +1,93 @@
+# Overview
+A custom, lightweight framework is used in this application. There are a few reasons a custom framework was built and we know it adds a bit of complexity, so what follows is a few reasons behind the decision. First the most desirable goal was a stable system. The influence of how funding can incentivize the building of software at the expense of maintaining software meant choosing an approach that offered some consistency over time. This translated to focusing on type safety. Things need to be strongly typed! Since most JS modules don't deal with types and change frequently, an argument could be made for minimizing external dependencies. Also, the reality of many developers working on the codebase created a desire for consistency to manage technical debt...the tradeoff with a prescriptive design approach is more initial effort for less chance of features being broken over time.
+
+## Architecture
+ELM architecture is similar to the custom framework, though one is a language and the other is simply TypeScript.
```

Comment:
FYI "ELM" should be "Elm." I think it was named after the tree species of the same name.

Reply:
resolved in https://github.com/bcgov/digital_marketplace/pull/330/commits/a633c3c3bb2fc8969ceadbd1e3f96160bcf8c7ef